src/components/form/contacts/index.tsx

```diff
@@ -40,18 +57,59 @@ const ContactsForm = ({ toggleModal, initialValues, isEdit, apiUrl }: IProps) =>
         await dispatch(updateContactAsync({ id: body.id, data: body }));
       } else {
         const body = { ...data } as IContacts;
-        await dispatch(createContactAsync({ apiUrl, data: body }));
+        await dispatch(createContactAsync({ apiUrl, data: body, user: userStore }));
       }
       toggleModal();
     } catch (error) {
       Notify('Something went wrong');
     }
   };
 
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const fileInput = event.target;
+    if (fileInput.files && fileInput.files.length > 0) {
+      setFileName(fileInput.files[0].name);
+    } else {
+      setFileName(null);
+    }
+  };
+
+  const getHelperComponent = () => {
+    switch (true) {
+      case !!fileName:
+        return <HelperImageText>{fileName}</HelperImageText>;
+      case !!errors['image']:
+        return <HelperText>Image is required</HelperText>;
+      default:
+        return <EmptyHelperText>.</EmptyHelperText>;
+    }
+  };
   return (
     <>
       <StyledHeading>Add contacts</StyledHeading>
       <form onSubmit={handleSubmit(handleContactSubmit)}>
+        <label htmlFor='icon-button-photo'>
```

Comment:
Let's move img uploader to the separate component

Reply:
fix: moved to a separate component